lib/src/main/java/com/auth0/android/lock/internal/configuration/Connection.java

```diff
@@ -28,6 +29,31 @@ private Connection(@NonNull String strategy, Map<String, Object> values) {
         this.name = name;
         this.values = values;
         parseUsernameLength();
+        parsePasswordComplexity();
+    }
+
+    private void parsePasswordComplexity() {
+        int policy = PasswordStrength.NONE;
+        String value = valueForKey("passwordPolicy", String.class);
+        if ("excellent".equals(value)) {
```

Comment:
Is a `switch` a better fit here? Seems like checking the same thing over and over is duplicative.